src/Member/MemberUtil.php

```diff
@@ -172,4 +179,43 @@ public function findOrCreate(string $email)
 
         return $member;
     }
+
+    /**
+     * @return \Contao\Model[]|null
+     */
+    public function getActiveGroups(int $memberId, array $options = [])
+    {
+        if (!$memberId) {
+            return null;
+        }
+
+        if (null === ($memberModel = $this->modelUtil->findModelInstanceByIdOrAlias('tl_member', $memberId, $options))) {
+            return null;
+        }
+
+        if (null === ($groups = StringUtil::deserialize($memberModel->groups, true))) {
```

Comment:
StringUtil::deserialize can never be null. If the second parameter is `true`, in any case the fallback is `[]` (empty array). Please remove the check hence.